ui/lib/ldap/addon/components/accounts-checked-out.hbs

```diff
@@ -0,0 +1,70 @@
+<Hds::Card::Container @hasBorder={{true}} class="has-padding-l" ...attributes>
+  <h3 class="is-size-4 has-text-weight-semibold">Accounts checked out by this token</h3>
+
+  <p class="has-text-grey">The accounts that are currently on lease with this token</p>
```

Comment:
Missing a period. It's not a title or placeholder text.
I'm sorry, this review is annoying me. Feel free to curse me.